src/encoded/static/components/PageTitle.js

```diff
@@ -154,8 +156,8 @@ const StaticPageTitle = React.memo(function StaticPageTitle(props){
 
 /** Based on 4DN content views & metadata, to be updated re: CGAP */
 const GenericItemPageTitle = React.memo(function GenericItemPageTitle(props){
-    const { context, schemas, alerts, href, session } = props;
-    const itemTitle = object.itemUtil.getTitleStringFromContext(context);
+    const { context, schemas, alerts, href, session, className } = props;
```

Comment:
Codacy found an issue: ['className' is assigned a value but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5793828)